core/templates/pages/email-dashboard-pages/email-dashboard-page.component.spec.ts

```diff
@@ -166,7 +166,7 @@ describe('Email Dashboard Page', function() {
   });
 
   describe('recheckStatus', function() {
-    it('should get existing query from backend to check its content',
+    it('should reckeck status when getting next page of queries',
```

Comment:
status of what?

Reply:
I actually don't know since the code and template do not give any information about which status is being rechecked.

Reply:
Can you let me know what you have understood? I can help you figure it out...

Reply:
I've updated it. But, what I understand is that recheck status gets again the page's query to check again the data.